### PR TITLE
doc: update docs about helm charts

### PR DESCRIPTION
**Pull request by hhyasdf**
Pull Request Description
---

### Special notes for reviews

### CODE REVIEW

File: charts/hybridnet/README.md [hunk 1]
```diff
@@ -1 +1,59 @@
...
+```shell
+helm install hybridnet hybridnet/hybridnet -n kube-system
+```
+The command deploys hybridnet on the Kubernetes cluster in the default configuration.
+
+_See [configuration](#configuration) below._
+
+_See [helm install](https://helm.sh/docs/helm/helm_install/) for command documentation._
+
+## Upgrading Chart
```
Comment (mars1024) [inline charts/hybridnet/README.md:31]: typo: Upgrade

File: charts/hybridnet/README.md [hunk 2]
```diff
@@ -1 +1,59 @@
...
+After deploying hybridnet with the default configuration, you can change the default network type any 
+time with these commands:
+
+```shell
+# Change default network type to Underlay
+helm upgrade hybridnet hybridnet/hybridnet -n kube-system --set advancedSettings.defualtNetworkType=Underlay
+```
+
+Of course, if you want to change your container network to use Underlay as default network type, you should
+apply some Underlay _Network/Subnet_ CR resources first.
```
Comment (mars1024) [inline charts/hybridnet/README.md:59]: typo: firstly

File: charts/hybridnet/README.md (hunk 1 above)
Comment (hhyasdf) [reply charts/hybridnet/README.md]: fixed.

File: charts/hybridnet/README.md (hunk 2 above)
Comment (hhyasdf) [reply charts/hybridnet/README.md]: fixed.

**Review by mars1024 — APPROVED** [review]
/lgtm

**Comment by hhyasdf** [comment]
Related to #121

**Pull request merged**